README.md

```diff
@@ -36,48 +37,63 @@ bundles of each MFE and simply serve a small index.html file through caddy.
+- Communications: [`eduNEXT/frontend-app-communications@ednx-release/olmo.master`](https://github.com/eduNEXT/frontend-app-communications/tree/ednx-release/olmo.master)
+
+In addition, we need to use a custom version of `tutor-mfe` that includes the patch
+`mfe-dockerfile-production-final` introduced in [`overhangio/tutor-mfe#179`](https://github.com/overhangio/tutor-mfe/pull/179).
 
 ### Building Caveats
 
-At the moment each MFE image will be tied to a specific CDN endpoint. As a result,
-it isn't possible to reuse the MFE image between environments (namely Staging and
-production). You will have to rebuild the image in the following scenarios:
+The value of `publicPath` is fixed at build time, therefore it's not possible to reuse
+the MFE docker image if we set it's value to a specific CDN endpoint.
+To avoid this limitation we use a script
+[`docker-entrypoint.sh`](tutormfe_extensions/templates/mfe/build/mfe/docker-entrypoint.sh)
+to dinamically change a placeholder value used in `publicPath` each time the container is
+started.
+
+We need to make sure our image is built using the placeholder `MFE_EXTENSIONS_PLACEHOLDER_STRING/app_name` as our `publicPath` (where `app_name` refers to the value of `MFE_MYMFE_MFE_APP.name`). In Nutmeg you configure `PUBLIC_PATH` through the `config.yml` file:
+
+```yaml
+MFE_LEARNING_MFE_APP:
+  env:
+    production:
+      PUBLIC_PATH: MFE_EXTENSIONS_PLACEHOLDER_STRING/learning
+      MFE_CONFIG_API_URL: /api/mfe_config/v1
+      APP_ID: learning
+  name: learning
+  port: 2000
+  repository: https://github.com/eduNEXT/frontend-app-learning.git
+  version: ednx-release/olmo.master
+```
+
+The learning MFE in eduNEXT fork also has support for the [`MFE_CONFIG_API`](https://github.com/openedx/edx-platform/blob/master/lms/djangoapps/mfe_config_api/docs/decisions/0001-mfe-config-api.rst) in Nutmeg, you must also include the variables `MFE_CONFIG_API_URL` and `APP_ID` in your
+`config.yml` while building the MFE to enable the API.
 
-1. You have a working production image and want to enable CDN support: You must
-   rebuild the image using the same parameters while setting the
-   `MFE_EXTENSIONS_CDN_URL` variable with your distribution.
-2. You modified your CDN endpoint and it now uses a different URL: Update the
-   value of `MFE_EXTENSIONS_CDN_URL` to the new URL.
-3. You made changes to the code of any of the MFEs in the image and want to
-   deploy a new version: If your configuration already had CDN support update
-   the other parameters as you usually do in your deployment process, if it
-   didn't have CDN support follow step #1.
+Once you enable the plugin and add the previous snippet to your configuration you can build
+an image with a _"dynamic"_ `publicPath`. By default the placeholder will be replaced by
+an empty string, which was the original value. In a Kubernetes environment it will be
+replaced with the value of `MFE_EXTENSIONS_CDN_URL`.
```

Comment:
```suggestion
Once you enable the plugin you can build an image with a _"dynamic"_ `publicPath`.
By default the placeholder will be replaced by an empty string, which was the original value.
In a Kubernetes environment it will be replaced with the value of `MFE_EXTENSIONS_CDN_URL`.
```